packages/react/src/icons/withIcon.tsx

```diff
@@ -23,6 +23,7 @@ export const withIcon = (
   const Icon = forwardRef<SVGSVGElement, ComponentPropsWithoutRef<"svg">>(
     (props, ref) => (
       <svg
+        aria-hidden
```

Comment:
similar comment here - let's just remove the `role="img"` instead of adding an `aria-hidden`